src/Lykke.AzureStorage/Blob/AzureBlob.cs

```diff
@@ -310,5 +310,17 @@ public string GetBlobUrl(string container, string key)
 
             return blockBlob.Metadata;
         }
+
+        /// <inheritdoc />
+        public async Task<BlobProperties> GetPropertiesAsync(string container, string key)
+        {
+            if (string.IsNullOrWhiteSpace(container) || string.IsNullOrWhiteSpace(key) || !await HasBlobAsync(container, key))
+                return null;
```

Comment:
May be will be better to return ```new BlobProperties()```? Do we need to distinguish when blob doesn't exists?

Reply:
We need to distinguish that blob does not exist. Because for calling code it would be unclear if it would get valid BlobProperties but with unfiilled fields. Caller would assume that this is the properties for the blob nad use them, which could lead to errors.